.github/scripts/install-libff.sh

```diff
@@ -9,23 +9,28 @@ if [ -f $HOME/.local/lib/libff"$EXT" ]; then
   exit 0
 fi
 
-git clone https://github.com/scipr-lab/libff --recursive
+if [ -d libff ]; then
+  echo "$(pwd)/libff" already exists! Using it instead of re-cloning the repo.
+else
+  git clone https://github.com/scipr-lab/libff --recursive
+fi
 cd libff
 git submodule init && git submodule update
 git checkout v0.2.1
 
 ARGS="-DCMAKE_INSTALL_PREFIX=$PREFIX -DWITH_PROCPS=OFF"
 CXXFLAGS=""
 if [ "$HOST_OS" = "macOS" ]; then
-  export LDFLAGS=-L/usr/local/opt/openssl/lib
-  export CPPFLAGS=-I/usr/local/opt/openssl/include
-  export CXXFLAGS=-I/usr/local/opt/openssl/include
-  ARGS="$ARGS -DOPENSSL_INCLUDE_DIR=/usr/local/opt/openssl/include/openssl -DCURVE=ALT_BN128 -DCMAKE_INSTALL_NAME_DIR=$PREFIX/lib"
+  OPENSSL_PREFIX=$(brew --prefix openssl)
```

Comment:
I noticed this returns `/opt/homebrew/opt/openssl@3` on my system, even when I only had `openssl@1.1` installed. From [the GH documentation](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md), the runners appear to have `openssl@1.1`. Do we need to take any extra care here to use a specific version?